boards/slstk3401a/Makefile.features

```diff
@@ -0,0 +1,14 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_adc
+FEATURES_PROVIDED += periph_gpio
+FEATURES_PROVIDED += periph_i2c
+FEATURES_PROVIDED += periph_rtc
+FEATURES_PROVIDED += periph_rtt
+FEATURES_PROVIDED += periph_spi
+FEATURES_PROVIDED += periph_timer
+FEATURES_PROVIDED += periph_uart
+
+# The board MPU family (used for grouping by the CI system)
+FEATURES_MCU_GROUP = cortex_m4_2
+
+include $(RIOTCPU)/efm32/Makefile.features
```

Comment:
just comparing to nucleos and I think if a '-' at the beginning of the line is required

Reply:
According to [this](https://stackoverflow.com/a/34395651) link, it should be prefixed if you don't care if the file exists. But I do :-)

I see that others have it, but I wonder why the `cpu` include is somewhat optional...

Reply:
Fixed it though.

Reply:
I do think the optional include in the other boards is a bug though.